packages/storybook/src/stories/grid-style.stories.tsx

```diff
@@ -1,219 +0,0 @@
-    age: 30,
-    registerDate: new Date(2011, 6, 16),
-    lastLoginDate: new Date(2020, 2, 14, 7, 30, 25),
-  },
-  {
-    id: 3,
-    lastName: 'Smith',
-    firstName: 'igor',
-    isRegistered: false,
-    age: 40,
-    registerDate: new Date(2016, 8, 1),
-  },
-  {
-    id: 4,
-    lastName: 'James',
-    firstName: 'clara',
-    isRegistered: true,
-    age: 40,
-    registerDate: new Date(2011, 1, 1),
-    lastLoginDate: new Date(2020, 2, 10, 15, 30, 25),
-  },
-  {
-    id: 5,
-    lastName: 'Bobby',
-    firstName: 'clara',
-    isRegistered: false,
-    age: null,
-    registerDate: new Date(2018, 0, 1),
-    lastLoginDate: new Date(2020, 5, 29, 18, 0, 25),
-  },
-  {
-    id: 6,
-    lastName: 'James',
-    firstName: null,
-    isRegistered: false,
-    age: 40,
-    registerDate: new Date(2013, 8, 16),
-    lastLoginDate: new Date(2019, 6, 4, 22, 36, 25),
-  },
-  { id: 7, lastName: 'Smith', firstName: '', isRegistered: true, age: 40 },
-];
-
-export const ColumnCellClass = () => {
-  const rows = React.useMemo(() => getRows(), []);
-  const cols = React.useMemo(() => getColumns(), []);
-  cols[3].cellClassName = () => clsx('age', 'shine');
-  cols[1].cellClassName = (params) => (!params.value ? 'unknown' : '');
-
-  return (
-    <div className="grid-container">
-      <DataGridPro rows={rows} columns={cols} />
-    </div>
-  );
-};
-export const ColumnHeaderClass = () => {
-  const rows = React.useMemo(() => getRows(), []);
-  const cols = React.useMemo(() => getColumns(), []);
-  cols[3].headerClassName = () => clsx('age', 'shine');
-
-  return (
-    <div className="grid-container">
-      <DataGridPro rows={rows} columns={cols} />
-    </div>
-  );
-};
-
-export const ColumnCellRenderer = () => {
-  const rows = React.useMemo(() => getRows(), []);
-  const cols = React.useMemo(() => getColumns(), []);
-  cols[5].renderCell = (params) => <IsDone value={!!params.value} />;
-
-  return (
-    <div className="grid-container">
-      <DataGridPro rows={rows} columns={cols} />
-    </div>
-  );
-};
-export const ColumnCellRendererWithPadding = () => {
-  const rows = React.useMemo(() => getRows(), []);
-  const cols = React.useMemo(() => getColumns(), []);
-  cols[5].renderCell = (params) => <IsDone value={!!params.value} />;
-
-  return (
-    <div className="grid-container" style={{ padding: 50 }}>
-      <DataGridPro rows={rows} columns={cols} />
-    </div>
-  );
-};
-export const TruncatedCellTextSnap = () => {
-  return (
-    <div style={{ height: 250, width: '100%' }}>
-      <DataGridPro
-        columns={[{ field: 'text' }]}
-        rows={[
-          {
-            id: 1,
-            text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam a accumsan lacus. Etiam faucibus fermentum ipsum. Integer bibendum diam quis est sagittis laoreet. Pellentesque finibus lacus vitae erat aliquet dignissim.',
-          },
-        ]}
-      />
-    </div>
-  );
-};
-export const MultilineCustomCellContentSnap = () => {
```

Comment:
Since the Argos CI step is broken, we don't have visibility on the scale of this, but here is one visual test that is gone. Will we miss these visual regression tests?

Reply:
![image](https://user-images.githubusercontent.com/3309670/188925697-1ee63fca-75cb-4a81-9f1d-95cf69b4e60a.png)

Here are all the stories with the `Snap` suffix
If the team wants to migrate some to a hidden doc example I can do it
I personally don't see any good candidate.

Reply:
For instance the most recent `PinnedColumnWithCheckboxSelection` has an equivalent in the doc:
https://mui.com/x/react-data-grid/column-pinning/#pinning-the-checkbox-selection-column

Reply:
> I personally don't see any good candidate.

Then I think that we are good

> to a hidden doc example

In the main repo, we store them here https://github.com/mui/material-ui/tree/master/test/regressions/fixtures (outside of the docs)

Reply:
Oh I think we had pickers fixtures that I did not migrate :smile: 
I'll have a look to maybe bring back this logic later